pkg/pods/pod_test.go

```diff
@@ -152,6 +156,10 @@ config:
 		launchableRootEnv, err := ioutil.ReadFile(filepath.Join(launchable.EnvDir(), "LAUNCHABLE_ROOT"))
 		Assert(t).IsNil(err, "should not have erred reading the launchable root env file")
 		Assert(t).AreEqual(launchable.InstallDir(), string(launchableRootEnv), "The launchable root path did not match expected")
+
+		enableBlamSetting, err := ioutil.ReadFile(filepath.Join(launchable.EnvDir(), "ENABLED_BLAMS"))
+		Assert(t).IsNil(err, "should not have erred reading custom env var")
+		Assert(t).AreEqual("5", string(enableBlamSetting), "The setting was wrong")
```

Comment:
probably should at least mention that it's a the launchable custom env var instead of a vague setting